code/src/componets/Form.js

```diff
@@ -0,0 +1,215 @@
+  const [inputRole, setInputRole] = useState('');
+  const [inputEnvironment, setInputEnvironment] = useState('');
+  const [inputExperience, setInputExperience] = useState(null);
+  const [inputTime, setInputTime] = useState(0);
+  const [formStage, setFormStage] = useState(0);
+  const [percent, setPercent] = useState(0);
+
+  const resetForm = () => {
+    setFormStage(0);
+    setInputName('');
+    setInputRole('');
+    setInputEnvironment('');
+    setInputExperience('');
+    setPercent(0);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setFormStage(formStage + 1);
+    const correctPercent = (formStage * 100) / 6;
+    setPercent(Number(correctPercent).toFixed(1));
+  };
+
+  return (
+    <main>
+      {formStage > 0 && <ProgressBar progress={percent} />}
+      <div className="title">
+        <p>
+          {formStage < 5 ? formStage : 5}
+          <span>→</span>
+        </p>
+
+        <h1>Mentorship Sign Up</h1>
+      </div>
+      <div className="image-box">
+        <picture>
+          <source media="(min-width:1042px)" srcSet={headerImgLarge} />
+          <source media="(min-width: 768px)" srcSet={headerImgMedium} />
+          <img src={headerImgSmall} alt="Sign with text: Do something Great" />
+        </picture>
+      </div>
+      {formStage >= 1 && (
+        <button className="reset-button" type="button" onClick={resetForm}>
+          Restart Form
+        </button>
+      )}
+      <form onSubmit={handleSubmit} required>
+        {formStage === 0 && (
+          <div>
+            <h2>
+              Welcome to the mentor sign up survey, thank you for showing
+              interest in becoming a mentor, press ok to continue in the form.
+              Please take a few minutes to fill in this survey to start your
+              journey in becoming a mentor.
+            </h2>
+          </div>
+        )}
+        {formStage === 1 && (
+          <div>
+            <h1>What is your name? *</h1>
+            <input
+              type="text"
+              value={inputName}
+              aria-required="true"
+              onChange={(e) => {
+                setInputName(e.target.value);
+              }}
+              required
+            />
+          </div>
+        )}
+        {formStage === 2 && (
+          <div>
+            <h1>What is your current roll? *</h1>
+            <select
+              onChange={(e) => {
+                setInputRole(e.target.value);
+              }}
+              value={inputRole}
+              required
+            >
+              <option value="">Pick a roll</option>
+              <option value="Developer">Developer</option>
+              <option value="HR">HR / Recruter</option>
+              <option value="Other">Other</option>
+            </select>
+          </div>
+        )}
+        {formStage === 3 && (
+          <div>
+            <h1>What time commitment can I make? /Hours a month/ *</h1>
```

Comment:
super minor nitpicky thing, the user has been asked stuff as a "you" before this, and now an "I". Might be nice to be consistent :)